assets/js/components/settings/SettingsEmailReporting.js

```diff
@@ -104,18 +118,29 @@ export default function SettingsEmailReporting( { loading = false } ) {
 					</div>
 				</Cell>
 			</Row>
-			{ isEnabled && (
-				<Row className="googlesitekit-settings-email-reporting__manage">
-					<Cell size={ 12 }>
-						<Link onClick={ handleManageClick }>
-							{ __(
-								'Manage email reports subscription',
-								'google-site-kit'
-							) }
-						</Link>
-					</Cell>
-				</Row>
-			) }
+			{ /* Show the introductory notice if notice is not dismissed and user is not already subscribed */ }
+			{ isEnabled &&
+				settings !== undefined &&
+				! isSubscribed &&
+				isDismissed === false && (
+					<EmailReportingCardNotice className="googlesitekit-settings-email-reporting__manage" />
```

Comment:
This should be wrapped with `<Row>` and `<Cell>`. Plus, we can move the `googlesitekit-settings-email-reporting__manage` class to the `Row` element and remove wrapping `div` in the notice.